PotatsoModel/Proxy.swift

```diff
@@ -86,7 +86,12 @@ open class Proxy: BaseModel {
         "verify_simple",
         "auth_simple",
         "auth_sha1",
-        "auth_sha1_v2"
+        "auth_sha1_v2",
+        "auth_sha1_v4",
+        "auth_aes128_md5",
+        "auth_aes128_sha1",
+        "auth_chain_a",
```

Comment:
Can we remove these two lines as you said it doesn't support `auth_chain_a` and `auth_chain_b` yet?

Reply:
Sorry. I forgot to move it in this file.